### ISSUE TITLE
Can't make release build (Android) due to error

### ISSUE THREAD

**Issue by leithalnajjar** [endigo/flutter_pdfview#236]
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':flutter_pdfview:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR:/Users/dimius555/Development/Flutter Projects/Mroad/build/flutter_pdfview/intermediates/merged_res/release/values/values.xml:132: AAPT: error: resource android:attr/lStar not found.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

**Comment by babysofttester** [comment]
Execution failed for task ':flutter_pdfview:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed

Same issue with me also

**Comment by upgrade-dev** [comment]
Same here
Flutter 3.7.9 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 62bd79521d (4 giorni fa) • 2023-03-30 10:59:36 -0700
Engine • revision ec975089ac
Tools • Dart 2.19.6 • DevTools 2.20.1

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.7.9, on Manjaro Linux 6.1.19-1-MANJARO, locale it_IT.UTF-8)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.1)
[✗] Chrome - develop for the web (Cannot find Chrome executable at google-chrome)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.
[✓] Linux toolchain - develop for Linux desktop
[✓] Android Studio (version 2021.3)
[✓] Connected device (2 available)
[✓] HTTP Host Availability

* What went wrong:
Execution failed for task ':flutter_pdfview:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR:/home/nixbi/.gradle/caches/transforms-3/36128c1351c9ec2b226115461677d9c1/transformed/core-1.7.0/res/values/values.xml:105:5-114:25: AAPT: error: resource android:attr/lStar not found.


* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

**Comment by Thelm76** [comment]
force 1.2.9 for now, this issue happened with 1.2.6, it was caused by the android depencency upgrade from beta2 to beta3

**Comment by charlesRmajor** [comment]
1.2.9 didn't work for me. Tried 1.2.5 and a few lower than that and they're all showing the same error.

**Comment by AlfaizCeras** [comment]
Same issue with me.

```
* What went wrong:
Execution failed for task ':flutter_pdfview:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR:/Users/alfaiz/Development/Ceras/Ceras_Mobile_v2/build/flutter_pdfview/intermediates/merged_res/release/values/values.xml:132: AAPT: error: resource android:attr/lStar not found.

```

**Comment by Alfaizkhan** [comment]
@leithalnajjar 
Temporary this will work for you.

```
  flutter_pdfview:
    git:
      url: https://github.com/Alfaizkhan/flutter_pdfview.git
      ref: 535077f5297c7027fc0fb862abb1cbf2ac9c8e6f
```

**Comment by Thelm76** [comment]
@endigo could you please update the documentation to add the needed requirements for last version ?

**Comment by jacky4631** [comment]
1.2.9 work for me, use flutter_pdfview: 1.2.9,don't contains ^,then flutter clean/build

**Comment by bishal-rumba007** [comment]
@jacky4631  do you mean i should inclue ^ infront of 1.2.9 or i should not include this?

**Comment by ameenbadri7** [comment]
> 1.2.9 work for me, use flutter_pdfview: 1.2.9,don't contains ^,then flutter clean/build

this worked for me ..thanks all.

**Comment by devmgs** [comment]
for the time being, it is not resolved to add this to pubspec
```
dependency_overrides:
  # Keep until https://github.com/endigo/flutter_pdfview/issues/236 is resolved.
  flutter_pdfview: 1.2.9
```

**Comment by bishal-rumba007** [comment]
I have come across same issue again.
I added 1.2.9 a week ago and it worked but its giving similar error

**Comment by devmgs** [comment]
need to be merged with #240

**Comment by endigo** [comment]
https://stackoverflow.com/a/69050529
